### PR TITLE
Tweaking UI styles

### PR DESCRIPTION
**Pull request by jeffbaumes**
@manthey let me know if these are sane changes. Mostly removed custom color styling and let Bootstrap do its standard mostly-grayish thing. I am setting the width of each component directly so collapsing does not change the width - could not think of a better way to achieve this.

### CODE REVIEW

File: client/js/map.js [hunk 1]
```diff
@@ -16,11 +16,11 @@
 geoapp.TileSets = {
     mapquest: {
         url: 'http://otile1.mqcdn.com/tiles/1.0.0/map/',
-        credit: 'Tiles Courtesy of <a href="http://www.mapquest.com/">MapQuest</a>'
+        credit: 'Tiles courtesy of <a href="http://www.mapquest.com/">MapQuest</a>'
```
Comment (manthey) [inline client/js/map.js]: It probably doesn't matter, but I copied the capitalization from MapQuest's developer's page: http://developer.mapquest.com/web/products/open/map.

File: client/stylesheets/controls.styl [hunk 2]
```diff
@@ -7,32 +7,32 @@
...
 
+.ga-main-ui
+  width 335px
+
 .ga-main-ui,.ga-results-ui
   user-select none
   position absolute
   .panel
-    background none
+    background-color rgba(255,255,255,0.75)
```
Comment (manthey) [inline client/stylesheets/controls.styl]: This background works well with the satellite map, but appears hard to read (to me) on Stamen Toner.  Perhaps something like #EEE with no opacity would be better?

File: client/stylesheets/controls.styl [hunk 3]
```diff
@@ -7,32 +7,32 @@
...
     padding 3px 5px
-    background-color rgba(255,128,0,0.75)
   .panel-default
     border 0
   .panel-body
     padding 5px
-    background-color rgba(255,160,160,0.75)
   .panel-group
-    margin-bottom 1px
+    margin 10px
```
Comment (manthey) [inline client/stylesheets/controls.styl]: I liked the smaller gaps between panels and around the edges...  It left more room in the center.  I don't feel strongly about it, but I figured I'd comment.

File: client/stylesheets/controls.styl [hunk 4]
```diff
@@ -210,7 +202,7 @@
     width 100%
     max-height 56px
     line-height 16px
-    overflow hidden
+    overflow scroll
```
Comment (manthey) [inline client/stylesheets/controls.styl]: How about

```
overflow-x hidden
overflow-y scroll
```

File: client/stylesheets/graph.styl [hunk 5]
```diff
@@ -3,7 +3,9 @@
 #ga-graph-page
   left 0
   right inherit
+  bottom 30px
```
Comment (manthey) [inline client/stylesheets/graph.styl]: Good.  I shouldn't have covered over the credits.

File: client/stylesheets/controls.styl [hunk 6]
```diff
@@ -151,8 +146,6 @@
       resize vertical
       table
         font-size 12px
-        background rgba(232, 255, 255, 0.75)
-        color black
```
Comment (manthey) [inline client/stylesheets/controls.styl:155]: Can we leave the `color black`?  Bootstrap defaults to `#333`, which looks washed out to me.

File: client/stylesheets/controls.styl (hunk 4 above)
Comment (jeffbaumes) [inline client/stylesheets/controls.styl]: Where does `overflow-x` come into play? Really long words?

Comment (manthey) [inline client/stylesheets/controls.styl]: Yes.

On Mon, May 18, 2015 at 2:17 PM, Jeffrey Baumes notifications@github.com
wrote:

> In client/stylesheets/controls.styl
> https://github.com/XDATA-Year-3/geoapp/pull/94#discussion_r30533313:
> 
> > @@ -210,7 +202,7 @@
> >      width 100%
> >      max-height 56px
> >      line-height 16px
> > -    overflow hidden
> > -    overflow scroll
> 
> Where does overflow-x come into play? Really long words?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/XDATA-Year-3/geoapp/pull/94/files#r30533313.

## 

David Manthey
R&D Engineer
Kitware Inc.
(518) 881-4439

Comment (manthey) [inline client/stylesheets/controls.styl]: For browsers that show fixed scroll bars, the horizontal scroll bar eats up
a lot of vertical space, so I'd rather hide it.

On Mon, May 18, 2015 at 2:17 PM, David Manthey david.manthey@kitware.com
wrote:

> Yes.
> 
> On Mon, May 18, 2015 at 2:17 PM, Jeffrey Baumes notifications@github.com
> wrote:
> 
> > In client/stylesheets/controls.styl
> > https://github.com/XDATA-Year-3/geoapp/pull/94#discussion_r30533313:
> > 
> > > @@ -210,7 +202,7 @@
> > >      width 100%
> > >      max-height 56px
> > >      line-height 16px
> > > -    overflow hidden
> > > -    overflow scroll
> > 
> > Where does overflow-x come into play? Really long words?
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/XDATA-Year-3/geoapp/pull/94/files#r30533313.
> 
> ## 
> 
> David Manthey
> R&D Engineer
> Kitware Inc.
> (518) 881-4439

## 

David Manthey
R&D Engineer
Kitware Inc.
(518) 881-4439

File: client/stylesheets/controls.styl (hunk 3 above)
Comment (jeffbaumes) [inline client/stylesheets/controls.styl]: I'll split the difference with 5px

File: client/stylesheets/controls.styl (hunk 2 above)
Comment (jeffbaumes) [inline client/stylesheets/controls.styl]: I have a change with standard white opaque background with the original gray outline - I think it looks decent

File: client/js/map.js (hunk 1 above)
Comment (jeffbaumes) [inline client/js/map.js]: I'll back this out - keep MapQuest happy.

File: client/stylesheets/controls.styl (hunk 6 above)
Comment (jeffbaumes) [inline client/stylesheets/controls.styl:155]: With opaque white background (the intended background from Bootstrap) I think `#333` looks good.

File: client/stylesheets/controls.styl (hunk 4 above)
Comment (jeffbaumes) [inline client/stylesheets/controls.styl]: Ok this is fixed in latest version.

**Comment by manthey** [comment]
I've just pushed a few adjustments for dealing with the change from 10 to 5 pixel spacing and the graph.  Also, made the instagram overlay not have a transparent background.  If you don't like it, change it back.

**Comment by jeffbaumes** [comment]
I like your changes - I put one more set of changes relating to instagram overlay header. Ready for final review in case I messed something else up.

**Comment by manthey** [comment]
Just one minor change.  The date fields were wrapping to take up two lines in the control panels.  I widened the control panels slightly.  Otherwise, LGTM.

**Comment by jeffbaumes** [comment]
Good catch. Your width fix works on my browser also. LGTM too, so merging.

**Pull request merged**